infra/aws/terraform/prow-build-cluster/variables.tf

```diff
@@ -37,6 +37,14 @@ variable "deploy_kubernetes_resources" {
   nullable    = false
 }
 
+# This variable defines if this cluster is used as a Prow build cluster.
+variable "prow_build_cluster" {
```

Comment:
I think `is_prod` or `is_canary` is easier to grasp for code reader.

Reply:
As discussed on slack, this is only temporary state. We want to add canary cluster to prow and remove this var soonish.

Reply:
We discussed this and we'll keep prow_build_cluster as it is because we plan on removing it later.